assets/js/mbta.ts

```diff
@@ -2762,27 +2811,12 @@ const stationConfig: {
         },
       },
       {
-        id: 'GLEC',
+        id: 'SLEC',
         name: 'Lechmere',
-        zonePositions: {
-          left: ['n'],
-          center: [],
-          right: ['m'],
-        },
         zones: {
-          n: {
-            label: 'North',
-            modes: {
-              auto: true,
-              custom: true,
-              headway: false,
-              off: true,
-            },
-          },
           m: {
-            label: 'South',
```

Comment:
Should this include `label: 'Busway',` ?

Reply:
Hmm yeah probably a good idea. Although the UI does have an overarching **Busway.** label at the top of this page, 'Mezzanine' is kind of ambiguous in this context so Busway is probably clearer than that